src/v/redpanda/application.cc

```diff
@@ -733,10 +733,13 @@ void application::setup_public_metrics() {
             "cpu",
             {sm::make_gauge(
               "busy_seconds_total",
-              [] {
-                  return std::chrono::duration<double>(
-                           ss::engine().total_busy_time())
-                    .count();
+              [max_seen = ss::make_lw_shared<double>(0.)]() {
```

Comment:
This indirection thru a smart pointer is needed because in the metrics internals somewhere this lambda is captured inside another non-mutable lambda, making it const, so we can't declare this lambda mutable. We can't use unique_ptr because in the internals of metrics this lambda is used to construct a `std::function` and this requires the lambda to be copyable.